pymc/distributions/distribution.py

```diff
@@ -257,40 +255,28 @@ def signature(cls_or_self) -> None | str:
 
         return signature
 
-    @_class_or_instancemethod
-    @property
+    @_class_or_instance_property
     def ndims_params(cls_or_self) -> Sequence[int] | None:
-        """Number of core dimensions of the distribution's parameters."""
+        """Return number of core dimensions of the distribution's parameters."""
```

Comment:
Extraordinarily pedantic, but the original docstring was more correct since this is a property (it shouldn't be viewed as "returning" anything, it just *is*)

Reply:
Tell that to pre-commit not me  Yes pre-commit mandates English now!

Reply:
I think it didn't complain for property but doesn't understand a subclass of property like we have now

Reply:
Sure. Doesn't matter though, let's get it released.